templates/artist/includes/artist_detail_overview.html

```diff
@@ -4,140 +4,125 @@
-        <br />
-        <br />
-    {% endif %}
-    {% if artist.bio %}
+        <h4>AMOUNT RAISED</h4>
+        <h4 class="investing">${{ campaign.amount_raised|intcomma }}</h4>
+        <h5>OUT OF ${{ campaign.amount|intcomma }}</h5>
         <div class="break">
             <hr>
         </div>
-        <!-- Artist Bio -->
-        <h2>OUR STORY</h2>
-        <div class="artist-detail-padding">
-            {{ artist.bio.bio|markdown }}
-        </div>
-    {% endif %}
 
-    {% if artist.soundcloudplaylist_set.exists %}
+            {% if request.user.is_authenticated %}
+                <!-- Buy Shares -->
+                <button id="invest-button" class="button buy" type="button">Buy {{ campaign.default_name_shares }} Share{{ campaign.default_num_shares|pluralize }}</button>
+                <div class="invest-num-shares">
+                    <button id="remove-shares" class="secondary hollow button disabled shares" type="button" disabled>-</button>
+                    <input type="number" min="1" max="{{ campaign.num_shares_remaining }}" value="{{ campaign.default_num_shares }}" />
+                    <button id="add-shares" class="secondary hollow button" type="button">+</button>
+                </div>
+                <p>NUMBER OF SHARES</p> 
+                <div class="total-cost">
+                    <span data-tooltip aria-haspopup="true" class="has-tip [tip-right] [radius round]" title="${{ PERDIEM_FEE }} PerDiem + Credit Card Fee (2.9%+$0.30)">
+                        Total <a id="shares-price"></a>
+                    </span>
+                </div>
+            {% else %}
+                <h4> to invest in this project</h4>
+                <button class="login-box signup" data-open="login-modal">Login</button>
+                <button class="login-box signup" >Sign up</button>
+            {% endif %}
+    </div>
+    {% endwith %}
+
+        <!-- Music -->
+        {% if artist.soundcloudplaylist_set.exists %}
         <div class="break">
             <hr>
         </div>
-        <!-- Music -->
-        <h2>MUSIC</h2>
+        <!-- Genre -->
+        {% with genres=artist.genres.all %}
+            <h4>Genre{{ genres|length|pluralize }}: {{ genres|join:" / " }}</h4>
+        {% endwith %}
+        <h4>Location: {{ artist.location }}</h4>
         <div class="artist-detail-padding">
             {% for playlist in artist.soundcloudplaylist_set.all %}
                 <iframe width="100%" height="166" scrolling="no" frameborder="no" src="https://w.soundcloud.com/player/?url={{ playlist.playlist }}&color=ff5500"></iframe>
             {% endfor %}
         </div>
-    {% endif %}
+        {% endif %}
 
-    {% if campaign %}
-        {% if campaign.expense_set.all %}
+        <!-- Artist Bio -->
+        <h4>Bio</h4>
+        {% if artist.bio %}
+            <div class="artist-detail-padding">
+                {{ artist.bio.bio|markdown }}
+            </div>
+        {% endif %}
+        {% if campaign %}
```

Comment:
The `{% if campaign %}` block should come before the "WHAT DO YOU GET FOR INVESTING?" header.